airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -92,8 +93,9 @@ def secret_property_names_fixture():
 DATETIME_PATTERN = "^[0-9]{4}-[0-9]{2}-[0-9]{2}(T[0-9]{2}:[0-9]{2}:[0-9]{2})?$"
 
 
-# The connector fixture can be long to load, we have to increase the default timeout...
-@pytest.mark.default_timeout(5 * 60)
+# Running tests in parallel can sometime delay the execution of the tests if downstream services are not able to handle the load.
+# This is why we set a timeout on tests that call command that should return quickly, like spec
+@pytest.mark.default_timeout(ONE_MINUTE)
```

Comment:
Is it possible for us to know if we've set an option to run concurrently? I.e. set timeouts based on if we're running concurrently or not? 

No worries if not, just a suggestion :)

Reply:
I know the `pytest.xdist` exposes a `worker_id` fixture that can help us check **within the test execution** if we're running concurrently or not. But these timeout are not controlled by the test themselves but at the pytest orcherstration level. If we want to set the timeout more dynamically I think we'd have to change something [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/plugin.py#L47) where we tweak tests configuration and have access to fixture value. I think it might be early optimization at the moment. Let's consider this change if we observe more timeout-related problems.